lms/services/lti_outcomes.py

```diff
@@ -191,34 +191,38 @@ def _send_request(outcomes_request_params, pox_body):
         force_include_body=True,
     )
 
-    # Execute request and check HTTP status.
     try:
         response = requests.post(
             url=outcomes_request_params.lis_outcome_service_url,
             data=xml_body,
             headers={"Content-Type": "application/xml"},
             auth=oauth_client,
         )
+        # The following will raise ``requests.exceptions.HTTPError`` if
```

Comment:
Maybe too much commenting, but this threw me.

Reply:
I think this requests stuff should move into a separate generic requests-lib wrapper at some point